client/components/payment-activity/style.scss

```diff
@@ -0,0 +1,34 @@
+.wcpay-payments-activity {
+	&__card__body {
+		text-align: center;
+		padding: 32px 0 !important;
```

Comment:
It looks like we don't need this anymore, so I removed it altogether here 41228facd.

That said, I'm also using `!important` in other parts of the CSS because we can't access the component's styling. I was hoping to fix this upstream too...